xml/smtp_banners.xml

```diff
@@ -1636,7 +1636,16 @@ The system or service fingerprint with the highest certainty overwrites the othe
     <param pos="0" name="service.product" value="Perl"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
-  <fingerprint pattern="^([^ ]+) E?SMTP(?: (?:Service )?Ready\.?)?$" flags="REG_ICASE">
+<fingerprint pattern="Cellopoint E-mail Firewall (v\d+.\d+.\d+) Build (\d+) ready">
+	<description>Matches a Cellopoint Firewall catching the request</description>
+	<example> Cellopoint E-mail Firewall v3.9.11 Build 0730 ready </example>
```

Comment:
You should test that the fields are properly extracted here.  Without attributes in `example`, when this is run via `rspec`/etc, as written this will only test that the regular expression matches.

Something like:

```
<example service.version="3.9.11" service.version.version="0730">...</example>